db/migrate/20140707203534_location_add_katello_default.rb

```diff
@@ -1,6 +1,6 @@
 class LocationAddKatelloDefault < ActiveRecord::Migration
   def up
-    add_column :taxonomies, :katello_default, :boolean, :null => false, :default => true
+    add_column :taxonomies, :katello_default, :boolean, :null => false, :default => false
```

Comment:
We need to change this for the following reason. 'katello_default' would automatically get set to true for all locations that were in foreman before this migrate script ran. We need to have only one katello_default